radix-engine-stores/src/lib.rs

```diff
@@ -1,13 +1,12 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
+extern crate core;
```

Comment:
Why is this needed?

Reply:
whaaaaaaaat who put it here? Seriously, not me.
(either a weird "update from develop" conflict artifact, or some random IntelliJ "auto-fix" action 🤦 )